modules/vaos/spec/services/v2/appointment_service_spec.rb

```diff
@@ -92,11 +92,14 @@
                            match_requests_on: %i[method path query]) do
             VCR.use_cassette('vaos/v2/mobile_facility_service/get_facility_200',
                              match_requests_on: %i[method path query]) do
-              allow(Rails.logger).to receive(:info).at_least(:once)
-              response = subject.post_appointment(va_booked_request_body)
-              expect(response[:id]).to be_a(String)
-              expect(response[:local_start_time])
-                .to eq(DateTime.parse('2022-11-30T13:45:00-07:00'))
+              VCR.use_cassette('vaos/v2/mobile_facility_service/get_clinic_200',
```

Comment:
Since moving the merging of facility and clinic details to the service, several tests needed to use an additional cassette. The clinic ID from the appointment requests was also modified to match the one the clinic request.